### PR TITLE
[NFC] comments clean up in test classes.

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Also remove a few unused variables

Overview
----------------------------------------
Trivial cleanup only

### CODE REVIEW

**Comment by seamuslee001** [comment]
This looks fine to me @monishdeb @totten just some cleanup of comments

File: tests/phpunit/CRM/Mailing/BaseMailingSystemTest.php [hunk 1]
```diff
@@ -57,7 +55,7 @@
   public function setUp() {
     $this->useTransaction();
     parent::setUp();
-    CRM_Mailing_BAO_MailingJob::$mailsProcessed = 0; // DGW
```
Comment (totten) [inline tests/phpunit/CRM/Mailing/BaseMailingSystemTest.php]: For the curious, `DGW` stands for "*Dear God, Why... Why do I have to reset this silly static variable by hand?*"

**Comment by totten** [comment]
Yeah, looks safe, and tests pass.

File: tests/phpunit/CRM/Mailing/BaseMailingSystemTest.php (hunk 1 above)
Comment (eileenmcnaughton) [reply tests/phpunit/CRM/Mailing/BaseMailingSystemTest.php]: :-)

**Pull request merged**